ci/teams/rhtap-servicerelease/release-build.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  name: rhtap-build-rpa
```

Comment:
we'd also need a corresponding `rhtap-gitops-rpa`

Reply:
+1, I was waiting to move the workspace to the rh cluster and also get a successful build out.